de-lib/src/main/java/org/iplantc/de/diskResource/client/views/metadata/dialogs/SelectMetadataTemplateDialog.java

```diff
@@ -52,8 +52,15 @@
 
     private MetadataView.Presenter.Appearance appearance;
 
+    private String BASE_ID = "select_metadata_template";
```

Comment:
I think for consistency these IDs should probably be defined with the rest of the [Metadata ids](https://github.com/cyverse-de/ui/blob/f69465f48b2a62b5a4a179e0fc5c1006f884eec4/de-lib/src/main/java/org/iplantc/de/diskResource/share/DiskResourceModule.java) and should be camel case.